README.md

```diff
@@ -24,6 +24,7 @@ plugins:
 custom:
   client:
     bucketName: unique-s3-bucketname-for-your-website-files
+    distributionFolder: client/dist
```

Comment:
This is an optional configuration value. Can we change this to 
`# distributionFolder: client/dist`